src/System.Private.ServiceModel/src/System/ServiceModel/Channels/DnsCache.cs

```diff
@@ -1,17 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Collections.Generic;
-using System.Diagnostics;
 using System.Net;
-using System.Net.NetworkInformation;
 using System.Net.Sockets;
 using System.Runtime;
-using System.ServiceModel;
-using System.ServiceModel.Dispatcher;
-using System.Text;
 using System.Threading.Tasks;
 #if FEATURE_NETNATIVE
+using System.Collections.Generic;
+
```

Comment:
I can't tell from the PR -- is this in alphabetical order?  Or are we keeping NET Native usings all together and apart?